locales/en-US/editor.properties

```diff
@@ -136,6 +136,10 @@ DND_SUCCESS_UNZIP=Successfully unzipped <b>{0}</b>.
 # {0} will be replaced by a tar filename
 DND_SUCCESS_UNTAR=Successfully untarred <b>{0}</b>.
 
+# Image Viewer
+
```

Comment:
Nit: If you expect localizers to see the comment above, you’ll need to remove the empty line (otherwise the comment will be ignored by Pontoon for this string)